### ISSUE TITLE
Memory usage

### ISSUE THREAD

**Issue by qarkai** [rilian-la-te/vala-panel-appmenu#118]
Hi,
I installed https://aur.archlinux.org/packages/vala-panel-appmenu-xfce-git/ with following options:
```
_disable_mate=1
_disable_xfce=0
_disable_vala=1
_disable_budgie=1
_disable_unity_gtk_module=0

_opts=(
	-DCMAKE_INSTALL_PREFIX=/usr
	-DCMAKE_INSTALL_LIBDIR=lib
	-DCMAKE_INSTALL_LIBEXECDIR=lib
	-DMAKE_BOLD_APPNAME=ON
)
```
After couple of hours I noticed that plugin took 2.5GB of memory. Is it expected behavior?

**Comment by snippins** [comment]
@qarkai It happens a few times for me when I killed the panel with **killall**, sometimes the plugin are not killed and start to take up memory. Is this your case? If it is just kill any thing with "killall panel-*-" right after killing the panel.

**Comment by qarkai** [comment]
@snippins No, I haven't `killall`-ed anything. I've launched Chromium, KDevelop, xfce-terminal, Kate, Steam, Thunar if that matters. Maybe something else, I don't remember correctly.

**Comment by rilian-la-te** [comment]
AFAIK, not. This means something weird in GTK.

**Comment by snippins** [comment]
@rilian-la-te Did you just answer in the wrong thread :)?

**Comment by rilian-la-te** [comment]
No, it is correct answer. I think this is a bug in allocator or into underlying GTK tech.

**Comment by rilian-la-te** [comment]
We have very bad memory leaks now. Will look for it.

**Comment by qarkai** [comment]
It usually leaks when I use KDevelop. KDevelop has very long menu which with app's caption doesn't fit to my screen. Maybe that's the cause.

**Comment by rilian-la-te** [comment]
I plug some of the leaks, but some remains.

**Comment by baronleonardo** [comment]
I figured out that any Qt5 app make appmenu go crazy and increase everytime I switch to another app and return to it .. also I realized that the menu took 1 second to show up between apps switching

**Comment by rilian-la-te** [comment]
Appmenu version? About delay - it is a feature now.

**Comment by rilian-la-te** [comment]
If it is master - can you write a top logs?

**Comment by baronleonardo** [comment]
0.6.92-1

**Comment by rilian-la-te** [comment]
Try master, please.

**Comment by baronleonardo** [comment]
I think the problem was in `appmenu-gtk-module` as after update from version `0.5.4-1` into `0.6.92-1` the problem solved

**Comment by rilian-la-te** [comment]
It cannot be in appmenu-gtk-module, because it is not used on Qt apps.
